eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -2,15 +2,17 @@ parameters:
   PreSteps: []
   EnvVars: {}
   MaxParallel: 0
+  BuildInParallel: true
+  # Increase timeout so Event Hubs tests do not timeout (Windows runs for ~2h 35m, on average due to having two target platforms)
+  # https://github.com/Azure/azure-sdk-for-net/issues/5982
+  TimeoutInMinutes: 175
```

Comment:
A trip through the last few days' scheduled tests shows that EventHubs `Test` jobs usually last around 46 minutes.. I think this comment no longer applies. 

It might make sense to set this default back down to 60. Storage tests, on the other hand, seem to consistently reach the 175 minute timeout and it might be worth upping just `sdk/storage/test.yml`'s timeout parameter.

Reply:
I'm planning to increase storage to 5 hours in a separate PR.  Once storage tests get stable, we might be able to decrease the timeout.

Let me look at the other builds to see what's reasonable for the default timeout.  And I'll remove the comment.